test/parallel/test-util-inspect.js

```diff
@@ -235,3 +235,17 @@ if (typeof Symbol !== 'undefined') {
   assert.equal(util.inspect(subject, options), '[ 1, 2, 3, [length]: 3, [Symbol(symbol)]: 42 ]');
 
 }
+
+// test Set
+assert.equal(util.inspect(new Set), 'Set {}')
```

Comment:
I have that in a new commit yet to be pushed. Is it normal that jshint doesn't touch tests?